spec/model/post_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
```

Comment:
- Hey, great job with the test, however, the file fails to load and throws error due to how you specify the file path, kindly update it to `require_relative 'rails_helper'`

- After you've updated the file path, kindly fix the test for post, they are all failing.

![image](https://github.com/ngeelani48/Blogg-app/assets/59397159/461edc94-d00e-48c7-8e89-a64d5fa48844)

Reply:
Hi, @zieeco thanks for the review 🌷 

On my side, it was not showing any failures. I will try to fix and kindly if you can tell me which command have you used for tests. 👍  So that I can test according to that. 💭